core/src/main/resources/error/error-classes.json

```diff
@@ -153,6 +153,9 @@
     "message" : [ "The feature is not supported: %s" ],
     "sqlState" : "0A000"
   },
+  "UNSUPPORTED_GROUPING_EXPRESSION" : {
+    "message" : [ "grouping()/grouping_id() can only be used with GroupingSets/Cube/Rollup" ]
```

Comment:
I am thinking `GroupingSets/Cube/Rollup` is not easy to understand. Right now I cannot immediately recall what is grouping sets or cube or rollup. 

Is it possible to use other way to explain grouping()/grouping_id()? Like they can only be used in syntax a, b, c

Reply:
`GroupingSets/Cube/Rollup` are actually first class SQL syntax. So saying them in error message should be fine as people can quickly search and then know what `GroupingSets/Cube/Rollup` is.